booster-chat-app/src/entities/Message.ts

```diff
@@ -0,0 +1,18 @@
+import { UUID } from '@boostercloud/framework-types'
+import { UserProfile } from './UserProfile'
+
+export enum MessageType {
+  CHAT = 'chat',
+  JOIN = 'join',
+}
+
+export class Message {
```

Comment:
Is this working without the `@Entity` annotation?

Reply:
Whoops I missed that! Thank you!!